Makefile

```diff
@@ -15,6 +15,7 @@ vet:
 mocks:
 	mockgen -package mocks -destination go/mocks/filtering.go github.com/rode/grafeas-elasticsearch/go/v1beta1/storage/filtering Filterer
 	mockgen -package mocks -destination go/mocks/index.go github.com/rode/grafeas-elasticsearch/go/v1beta1/storage/esutil IndexManager
+	counterfeiter go/v1beta1/storage/esutil Client
```

Comment:
In a follow-up, we can move this to a [`go:generate`](https://github.com/maxbrunsfeld/counterfeiter#step-2b---add-counterfeitergenerate-directives) near or in the same file as the `Client` definition.